wg-xai/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  -
```

Comment:
We should include a reviewers/approvers group. At a minimum a list of people will be needed.

Reply:
I'll fix that.